crates/sui/src/validator_commands.rs

```diff
@@ -166,13 +166,21 @@ pub enum SuiValidatorCommand {
         gas_budget: Option<u64>,
     },
     /// Sui native bridge committee member registration
-    BridgeCommitteeRegistration {
-        /// Path to Bridge Authority Key file
+    #[clap(name = "register-bridge-committee")]
+    RegisterBridgeCommittee {
+        /// Path to Bridge Authority Key file.
         #[clap(long)]
         bridge_authority_key_path: PathBuf,
-        /// Bridge authority URL which clients collects action signatures from
+        /// Bridge authority URL which clients collects action signatures from.
         #[clap(long)]
         bridge_authority_url: String,
+        /// If true, only print the unsigned transaction and do not execute it.
+        /// This is useful for offline signing.
+        #[clap(long, default_value = "false")]
+        print_unsigned_transaction_only: bool,
```

Comment:
this flag is a bit verbose but I don't have a better suggestion, maybe `print_only` is good enough?

this is just nit, will leave it to you to decide.